dcos_installer/action_lib.py

```diff
@@ -530,15 +530,6 @@ def _add_prereqs_script(chain):
 yum_install xz
 yum_install ipset
 
-echo "Validating SELinux..."
```

Comment:
The check says: "Error if SELinux is enforcing".
This experiment removes it.
We need to change this to instead say "Error if SELinux is enforcing and mode is not targeted.".

To ship this, we will need to re-add this check, but add a check before `echo "Disabling enforcement..."`.

That line would look something like the following:

```
config=$(sestatus)
if [[ "$(getenforce)" == "Enforcing" ]]; then
  if [[ $config != *"Current mode:                   enforcing"* ]]; then
    echo "Disabling enforcement..."
    sudo setenforce 0
  fi
fi
```

Reply:
(that seems to work but uses the config, I'll make one which uses `sestatus`)